### PR TITLE
add equal employment opportunity commitment legal page

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- added /legal/equal-employment-opportunity-commitment, based on [this copy doc](https://docs.google.com/document/d/1_ExS3fRc8odw8extzVeZWkrZU78DekJ3Ei_dUOayGLI/edit)
- added a card for it on /legal index.

## QA

- visit https://ubuntu-com-11925.demos.haus/legal
- see that a link to the new page appears as a card
- follow the link, see that the content of the page matches the copy doc

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5787

### CODE REVIEW

**Comment by davegoddard42** [comment]
LGTM @sowasred2012 - a bit dry but I think that's right for this kind of page. I agree about using the cards on the homepage as the only navigation, with nothing added in the nav bar.

File: templates/legal/equal-employment-opportunity-commitment.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+{% block content %}
+<div class="p-strip is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1>Canonical Equal Employment Opportunity Commitment</h1>
+
+      <p class="p-text--small">In compliance with USA Code: 41 C.F.R. §§ 60-300.44(a); 60-741.44(a)</p>
+
+      <p>It is the policy of Canonical USA Inc. (the  “Company”) to provide equal employment opportunity to all employees and applicants without regard to race, religion, sex, gender identity, sexual orientation, national origin, physical or mental disability, status as a protected veteran, or any other characteristics protected by law.  This policy governs all terms and conditions of employment, including recruiting, hiring, training, promotion, demotion, transfer, layoff, compensation, discipline, and termination.</p>
```
Comment (petesfrench) [inline templates/legal/equal-employment-opportunity-commitment.html:14]:
```suggestion
      <p>It is the policy of Canonical USA Inc. (the  "Company") to provide equal employment opportunity to all employees and applicants without regard to race, religion, sex, gender identity, sexual orientation, national origin, physical or mental disability, status as a protected veteran, or any other characteristics protected by law.  This policy governs all terms and conditions of employment, including recruiting, hiring, training, promotion, demotion, transfer, layoff, compensation, discipline, and termination.</p>
```
Straighten the speech marks

**Comment by petesfrench** [comment]
@sowasred2012 There is a section at the bottom that feels a bit out of place of a legal document. Also the 'further reading' implies it is relevant content, which it is not
image

**Comment by sowasred2012** [comment]
@petesfrench that's a common footer amongst all legal pages

**Review by petesfrench — APPROVED** [review]
LGTM

**Pull request merged**